packages/atomic/src/components/commerce/product-template-components/atomic-product-price/atomic-product-price.pcss

```diff
@@ -0,0 +1,11 @@
+.display-grid {
```

Comment:
This makes the height of the product price always the same regardless of the presence of a promo price or not.